keras_hub/src/utils/keras_utils.py

```diff
@@ -104,3 +104,44 @@ def running_on_gpu():
         import torch
 
         return torch.cuda.is_available()
+
+def gpu_is_a100():
```

Comment:
We should name this in a way that will hold up over time. `gpu_supports_fused_attetion_op()` maybe.